src/Relude/Extra/CallStack.hs

```diff
@@ -27,6 +29,8 @@ that the caller function has 'HasCallStack' constraint. Otherwise, it returns
 >>> bar :: HasCallStack => String; bar = foo
 >>> bar
 "foo"
+
+@since 0.2.0
```

Comment:
Can we add `0.2.0` for the `callerName` as well? I've discovered that it was added in this PR: https://github.com/kowainik/relude/pull/57